Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/ProfileViewController.m

```diff
@@ -8,20 +8,65 @@
 
 #import "ProfileViewController.h"
 #import <Parse/Parse.h>
+#import "User.h"
 #import "AppDelegate.h"
 #import "SceneDelegate.h"
 #import "InitialLoginViewController.h"
 #import "Utils.h"
 
 @interface ProfileViewController ()
+@property (weak, nonatomic) IBOutlet UILabel *userNameLabel;
+@property (weak, nonatomic) IBOutlet UILabel *userUsernameLabel;
+@property (weak, nonatomic) IBOutlet UILabel *userEmailLabel;
+@property (weak, nonatomic) IBOutlet UIImageView *userProfilePicImageView;
+@property (weak, nonatomic) IBOutlet UISegmentedControl *personalNonprofitSegmentedControl;
+@property (nonatomic, strong) User *user;
+@property (weak, nonatomic) IBOutlet UIView *nonprofitView;
+@property (weak, nonatomic) IBOutlet UIView *personalView;
 
 @end
 
 @implementation ProfileViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    // Do any additional setup after loading the view.
+    self.user = [User currentUser];
+    self.userNameLabel.text = [NSString stringWithFormat:@"%@ %@", self.user.firstName, self.user.lastName];
+    self.userUsernameLabel.text = self.user.username;
+    self.userEmailLabel.text = self.user.email;
+    self.userProfilePicImageView.layer.cornerRadius = self.userProfilePicImageView.frame.size.width / 2;
+    [self.user.profilePicFile getDataInBackgroundWithBlock:^(NSData * _Nullable data, NSError * _Nullable error) {
+        if (error != nil) {
+            self.userProfilePicImageView.image = [UIImage imageNamed:@"PlaceholderProfilePic"];
+        } else {
+            UIImage *image = [UIImage imageWithData:data];
+            self.userProfilePicImageView.image = image;
+        }
+    }];
+    if (self.user.nonprofit == nil) {
+        self.personalNonprofitSegmentedControl.hidden = YES;
+        self.nonprofitView.hidden = YES;
```

Comment:
You can just move this out of the if/else if you want to set `self.nonprofitView.hidden = YES;` in both cases